data/text/text_4.asm

```diff
@@ -211,6 +211,19 @@ _CableClubNPCPleaseWaitText::
 	text_end
 
 _CableClubNPCLinkClosedBecauseOfInactivityText::
+	vc_patch Change_MSG
```

Comment:
```
_CableClubNPCLinkClosedBecauseOfInactivityText::
	vc_patch Change_MSG
if DEF(_RED_VC) || DEF(_BLUE_VC)
	text "Please come again!"
	done
	text_start
	text "sed because of"
	cont "inactivity."
else
	text "The link has been"
	line "closed because of"
	cont "inactivity."
endc
	vc_patch_end

	para "Please contact"
	line "your friend and"
	cont "come again!"
	done
```

Reply:
Resolved in 827f8e6